README.md

```diff
@@ -1,91 +1,39 @@
-# CleverHans (latest release: v3.1.0)
-
-Note: This is the final version of CleverHans v3, which supports TensorFlow 1, Python 3 and Python 2. In our version 4 update, we’re dropping support for TF1 and Python 2, and transitioning to JAX, PyTorch and TF2 on Python 3.
-
-This version was last tested with Python 3.5 and Tensorflow {1.8, 1.12} on Ubuntu 14.04.5 LTS (Trusty Tahr). You’re welcome to continue using it, but it’s no longer maintained.
+# CleverHans (latest release: v4.0.0)
 
 <img src="https://github.com/tensorflow/cleverhans/blob/master/assets/logo.png?raw=true" alt="cleverhans logo">
 
-[![Build Status](https://travis-ci.org/tensorflow/cleverhans.svg?branch=master)](https://travis-ci.org/tensorflow/cleverhans)
-[![Documentation Status](https://readthedocs.org/projects/cleverhans/badge/?version=latest)](https://cleverhans.readthedocs.io/en/latest/?badge=latest)
 
 This repository contains the source code for CleverHans, a Python library to
 benchmark machine learning systems' vulnerability to
 [adversarial examples](http://karpathy.github.io/2015/03/30/breaking-convnets/).
 You can learn more about such vulnerabilities on the accompanying [blog](http://cleverhans.io).
 
 The CleverHans library is under continual development, always welcoming
-[contributions](https://github.com/tensorflow/cleverhans#contributing)
+[contributions](https://github.com/cleverhans-lab/cleverhans#contributing)
 of the latest attacks and defenses.
-In particular, we always welcome help towards resolving the [issues](https://github.com/tensorflow/cleverhans/issues)
+In particular, we always welcome help towards resolving the [issues](https://github.com/cleverhans-lab/cleverhans/issues)
 currently open.
 
-## Major updates coming to CleverHans
+Since v4.0.0, CleverHans supports 3 frameworks: JAX, PyTorch, and TF2. In versions v3.1.0 and prior,
```

Comment:
Should we say that we will not strive to implement a feature in all 3 frameworks? Did we find a good way to deal with disparate features across frameworks?